### ISSUE TITLE
Invoke-Build produces excessive warning messages

### ISSUE THREAD

**Issue by virmitio** [coapp/coapp.powershell#5]
When running Invoke-Build, the following message is seen with extreme frequency in the console output:

```
WARNING: [0]:Warning [1]:[2]
```

**Comment by fearthecowboy** [comment]
Fixed in 1.23.207.0 -- To Update, use: 

``` powershell
# Update CoApp PowerShell Tools
Update-CoAppTools -KillPowerShells -Beta 
```

(Now actually says the Warning!)

**Comment by virmitio** [comment]
Confirmed fixed in 1.23.207
